src/game.py

```diff
@@ -43,31 +44,55 @@ def process_user_input(self):
         self.total_marks_on_board = self.board.count_marks(self.game_board)
         return self.game_board
 
+    def new_game(self):
+        self.total_marks_on_board = 0
+        self.game_board = Board().starter_board
+        self.get_formatted_board()
+        self.play_game()
+
     def get_winning_mark(self, total_marks_on_board):
         player = self.get_current_player(total_marks_on_board)
         if player == self.player_one:
             return self.player_two
         else:
             return self.player_one
 
+    def handle_winning_game(self, board):
+        winner = self.get_winning_mark(self.total_marks_on_board)
+        message.display_winner_message(self, winner)
+        self.repeat_game()
+
+    def handle_draw(self):
+        self.get_prompt(self.total_marks_on_board)
+        self.process_user_input()
+        self.total_marks_on_board = self.board.count_marks(self.game_board)
+        self.get_formatted_board()
+        self.play_game()
+
+    def repeat_game(self):
+        message.display_play_again(self)
+        self.play_again = False
+        if self.user_interface.get_play_again_user_input() == "Y":
+            self.play_again = True
+        while self.play_again:
+            self.new_game()
```

Comment:
This should be pulled out into a larger game loop.  Nesting the loop this far in is trouble.